src/main/java/com/stripe/net/RequestOptions.java

```diff
@@ -93,9 +106,12 @@ public int hashCode() {
     int result = apiKey != null ? apiKey.hashCode() : 0;
     result = 31 * result + (clientId != null ? clientId.hashCode() : 0);
     result = 31 * result + (stripeVersion != null ? stripeVersion.hashCode() : 0);
+    result = 31 * result + (stripeVersionOnBehalfOf != null ? stripeVersionOnBehalfOf.hashCode()
+        : 0);
     result = 31 * result + (idempotencyKey != null ? idempotencyKey.hashCode() : 0);
-    result = 31 * result + readTimeout;
+    result = 31 * result + (stripeAccount != null ? stripeAccount.hashCode() : 0);
```

Comment:
`stripeAccount` is added, although not related to this PR. I think it was unintentionally left out

Reply:
We could probably replace these manual implementations with Lombok's `@EqualsAndHashCode`.

Reply:
whoops, sorry i missed this and merged to my branch without it. Will update in subsequent PR